### ISSUE TITLE
Compilation error in generated code when there is a table called OVERRIDE

### ISSUE THREAD

**Issue by ainslec** [jOOQ/jOOQ#10498]
### Expected behavior and actual behavior:

The expected behaviour is that the generated code will compile (when JOOQ and Java 8 is on the classpath).

The actual behaviour is that multiple "Override is not an annotation type" errors are present in the code, as JOOQ generates code that imports the Override table, which will then override when the "Override" annotation is loaded from.

### Steps to reproduce the problem (if possible, create an MCVE: https://github.com/jOOQ/jOOQ-mcve):

Use code generator on a H2 database set up with the following tables:

This will probably work with any database, as the problem is not with the code of JOOQ, but  (appears to be) the code generator.

```
DROP TABLE IF EXISTS override;

CREATE TABLE override (
   field1      VARCHAR (8) NOT NULL,
   field2      VARCHAR (30) NOT NULL
);

DROP TABLE IF EXISTS my_table_1;

CREATE TABLE my_table_1 (
   field1      VARCHAR (8) NOT NULL,
   field2      VARCHAR (30) NOT NULL
);

DROP TABLE IF EXISTS my_table_2;

CREATE TABLE my_table_2 (
   field1      VARCHAR (8) NOT NULL,
   field2      VARCHAR (30) NOT NULL
);
```

### Versions:

- jOOQ: 3.13.4
- Java: Java 8
- Database (include vendor): H2
- OS: Windows 10 Pro
- JDBC Driver (include name if inofficial driver): org.h2.Driver

### Suggested remedy

If a table called "Override" is generated, then fully qualify all code generations for "@Override" as "@java.lang.Override". Or fully qualify all of them anyway (the nuclear option).

**Comment by lukaseder** [comment]
Thanks a lot for your report.

Normally, `<fullyQualifiedTypes/>` can be used to work around this:
https://www.jooq.org/doc/3.13/manual/code-generation/codegen-advanced/codegen-config-generate/codegen-generate-fully-qualified-types/

But it seems that the configuration is not applied for the `@Override` annotation

**Comment by lukaseder** [comment]
Fixed in jOOQ 3.14.0. Backport scheduled for 3.13.5 (#10500)

**Comment by lukaseder** [comment]
This fix will make sure `<fullyQualifiedTypes/>` works for `java.lang.Override` as well. But it would be nice if users didn't have to manually provide such a regular expression, and jOOQ could auto-detect all potential collisions: https://github.com/jOOQ/jOOQ/issues/10501
